### PR TITLE
Added more resources

### PR DESCRIPTION
**Pull request by saintdle**
Added more blogs, new videos section,

### CODE REVIEW

**Review by mtardy — COMMENTED** [review]
Thanks a lot, just a little nit!

File: docs/content/en/docs/resources/_index.md [hunk 1]
```diff
@@ -28,7 +28,17 @@ description: "Additional resources to learn about Tetragon"
 
 [Detecting and Blocking log4shell with Isovalent Cilium Enterprise](https://isovalent.com/blog/post/2021-12-log4shell) - Jed Salazar, 2021
 
+[Strengthen your system with Tetragon's eBPF-based Security Observability and Runtime Enforcement capabilities](https://b-nova.com/en/home/content/strengthen-your-system-with-tetragons-ebpf-based-security-observability-and-runtime-enforcement-capabilities) - Stefan Welsch, 2022
+
+[Can I Use Tetragon without Cilium? Yes!](https://isovalent.com/blog/post/can-i-use-tetragon-without-cilium-yes/) - [Dean Lewis](https://twitter.com/saintdle), 2023
```
Comment (mtardy) [inline docs/content/en/docs/resources/_index.md:33]: Sorry just a nit, can you put in the order of more recent to the least recent, as we do with conference talks just above? :)

Comment (mtardy) [reply docs/content/en/docs/resources/_index.md:33]: with the two above from 2021

**Comment by saintdle** [comment]
> Thanks a lot, just a little nit!

done

**Comment by mtardy** [comment]
> > Thanks a lot, just a little nit!
> 
> done

Thanks a lot, can you squash those two commits?

File: docs/content/en/docs/resources/_index.md [hunk 2]
```diff
@@ -24,11 +24,21 @@ description: "Additional resources to learn about Tetragon"
 
 ### Blog posts
 
+[Can I Use Tetragon without Cilium? Yes!](https://isovalent.com/blog/post/can-i-use-tetragon-without-cilium-yes/) - [Dean Lewis](https://twitter.com/saintdle), 2023
```
Comment (mtardy) [inline docs/content/en/docs/resources/_index.md:27]:
```suggestion
[Can I Use Tetragon without Cilium? Yes!](https://isovalent.com/blog/post/can-i-use-tetragon-without-cilium-yes/) - [Dean Lewis](https://twitter.com/saintdle), 2023
```
For some reason, Twitter is making the link checker timeout and I would not rely on Twitter from now on... Is it okay if we remove this link?
```
"url": "https://twitter.com/saintdle",
        "status": {
          "text": "Timeout"
}
```

Comment (saintdle) [reply docs/content/en/docs/resources/_index.md:27]: done

Comment (mtardy) [reply docs/content/en/docs/resources/_index.md:27]: I think link is still there https://github.com/cilium/tetragon/compare/63f2051b9d32b8ecf47566b835759d09435a9df1..302b6b6978651e4bdae17d4759227860363e43f2 you just changed your commit message. I think you might have forgot to `git add` sorry I skipped that in the process. You need a blank line before the "Signed-off" thingie as well.

**Comment by mtardy** [comment]
Hey do you need help? Sorry I don't want to make the process tedious for such a small modification, I can help you if needed.

**Pull request closed without merging**